### DISCUSSION TITLE
sensor.ecodan_dhw_consumed And sensor.ecodan_heating_consumed

### DISCUSSION THREAD

**Discussion by bjornenvall** [gekkekoe/esphome-ecodan-hp#22] [General]
Hi, i want to use the 2 sensors in the energy tab. But it doesn’t seem to work because they are not only increasing. Do you have any idea how to fix this? 
![IMG_1957](https://github.com/gekkekoe/esphome-ecodan-hp/assets/34765869/d024d4e1-3d12-4c22-8cbe-19a965bb9f1e)

**Comment by bwduncan** [comment]
Can you use a utility meter with delta values = True?

**Comment by gekkekoe** [comment]
the values are reported once a day. But to be honest they are far from accurate. It's a nice indication. I actually have a certified kwh meter and can compare the results over 24h.

But If you want to go with the ecodan numbers, then I think you can use a utility meter as @bwduncan suggest.
I did hear from ppl that when you connect a pulse kwh meter to the ecodan, the results becomes accurate (but you can read the pulse kwh meter directly...)

**Comment by gekkekoe** [comment]
see https://github.com/gekkekoe/esphome-ecodan-hp/discussions/35#discussioncomment-10412925
